feat: keychain component

baumstern opened the pull request:
fixes #32.

baumstern committed:
feat: implements key

baumstern committed:
test: add test of key

baumstern committed:
feat: implement mem store

baumstern committed:
test: add test of mem store

baumstern committed:
fix: apply goimports

baumstern committed:
feat: change interface to check error

baumstern committed:
fix: reflect interface change of keychain package

hea9549 commented on keychain/mem/store.go:
```diff
@@ -1,15 +1,34 @@
+	"github.com/DE-labtory/zulu/keychain"
+)
 
 type MemStore struct {
```
`mem` 이 패키지네임이니까 `mem.KeyStore`, `mem.NewKeyStore`같은 형식을 취하게 되야할것같습니다~
`MemStore` -> `KeyStore`로 변경 어떤가요?

baumstern replied:
반영 완료!

baumstern committed:
refactor: do not repeat a package name in function signature

apply the [review](https://github.com/DE-labtory/zulu/pull/33#discussion_r471877793) from @hea9549

baumstern committed:
refactor: reflect the review
